### DISCUSSION TITLE
search box goes invisible when clicked or when opening start menu

### DISCUSSION THREAD

**Discussion by Rezy774** [valinet/ExplorerPatcher#3134] [Q&A]
btw my version is win11 23H2 or 22631.3296

im using latest explorer patcher ver

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3107

please close this as this isn't planned to be fixed due to windows 11 24h2 removing the windows 10 start menu code entirely.
